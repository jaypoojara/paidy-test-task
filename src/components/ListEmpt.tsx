import React from "react";
import { Text, StyleSheet, View } from "react-native";
import { colors } from "../types/colors";

const ListEmpt = () => {
  return (
    <View style={styles.emptyContainer}>
      <Text style={styles.emptyText}>ToDos not found!</Text>
    </View>
  );
};

export default ListEmpt;

// Styles for the components
const styles = StyleSheet.create({
  emptyContainer: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
  },
  emptyText: {
    fontSize: 16,
    fontWeight: "bold",
    color: colors.lable,
  },
});
