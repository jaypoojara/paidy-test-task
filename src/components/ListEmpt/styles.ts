import { StyleSheet } from "react-native";
import { Colors } from "../../types/colors";

// Styles for the components
const Styles = StyleSheet.create({
  emptyContainer: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
  },
  emptyText: {
    fontSize: 16,
    fontWeight: "bold",
    color: Colors.lable,
  },
});

export { Styles };
