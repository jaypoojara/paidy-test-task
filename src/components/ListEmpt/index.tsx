import React from "react";
import { Text, View } from "react-native";
import { Styles } from "./styles";

const ListEmpt = () => {
  return (
    <View style={Styles.emptyContainer}>
      <Text style={Styles.emptyText}>ToDos not found!</Text>
    </View>
  );
};

export default ListEmpt;
