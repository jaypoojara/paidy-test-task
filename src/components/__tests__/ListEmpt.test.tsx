import React from "react";
import { render } from "@testing-library/react-native";
import ListEmpt from "../ListEmpt";

describe("ListEmpt Component", () => {
  it("renders correctly", () => {
    const element = render(<ListEmpt />);
    expect(element).toMatchSnapshot();

    expect(element.getByText("ToDos not found!")).toBeDefined();
  });
});
