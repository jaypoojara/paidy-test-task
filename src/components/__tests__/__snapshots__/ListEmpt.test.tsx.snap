// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListEmpt Component renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "#7a7a7a",
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  >
    ToDos not found!
  </Text>
</View>
`;
